### PR TITLE
Bz1928398

### PR DESCRIPTION
**Pull request by berndfinger**
This will solve a few issues: All the issues which are already fixed in branch bz1891642, plus namespacing all variables (issue #151), plus removing trailing dots in all yml files (issue #162)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +162,19 @@ If you want the role to set the RHEL release to a certain fixed minor release (a
...
 
+### Minimum package check
+The following variable will make sure packages are installed at minimum required versions as defined in files `vars/*.yml`. Default is `yes`.
+```yaml
+sap_hana_preconfigure_min_package_check
+```
+
+### Perform a yum update
+If the following variable is set to `yes`, the role will run a `yum update` before performing configuration changes. Default is `no`. \
+*Note*: The outcome of a `yum update` depends on the managed node's configuration for sticky OS minor version, see the description of the release option in `man subscription-manager`. For SAP HANA installations, setting a certain minor version with `subscscription-manager release --set=X.Y` is a strict requirement.
```
Comment (richm) [inline README.md]: `subscription-manager release --set=X.Y`

**Comment by richm** [comment]
This looks ok - but a couple of good practices items for future PRs, not this one:

When doing long strings, use the yaml `|` and `>` https://yaml.org/spec/1.2/spec.html#Block for readability e.g.
```yaml
  paramname: |
    these
    are
    several
    lines
```
would be rendered as 
```
these
are
several
lines
```
and 
```yaml
  paramname: >
    this
    is
    one
    line
```
would be rendered as
```
this is one line
```

Another - use `service_facts` wherever possible instead of `command: systemctl status name`, and registering and parsing the output - same with `package_facts` vs. `command: dnf or yum`

**Review by richm — APPROVED** [review]
lgtm

**Comment by rhmk** [comment]
lgtm

**Pull request merged**